README.md

```diff
@@ -2,18 +2,18 @@
 
 [![Build Status](https://travis-ci.org/tom-dalton-fanduel/fdjangodog.svg?branch=master)](https://travis-ci.org/tom-dalton-fanduel/fdjangodog)
 
-A simple Django middleware for submitting timings and exceptions to Datadog.
+A simple Django middleware for submitting timings and exceptions to Datadog using statsd.
```

Comment:
Maybe we should renamed "statsd" here to "DogStatsD" to be 100% clear https://docs.datadoghq.com/guides/dogstatsd/